app/src/main/java/com/asfoundation/wallet/fingerprint/FingerprintFragment.kt

```diff
@@ -0,0 +1,78 @@
+package com.asfoundation.wallet.fingerprint
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.asf.wallet.R
+import com.asfoundation.wallet.repository.PreferencesRepositoryType
+import com.jakewharton.rxbinding2.view.RxView
+import dagger.android.support.DaggerFragment
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import kotlinx.android.synthetic.main.fragment_fingerprint_layout.*
+import kotlinx.android.synthetic.main.fragment_permissions_list_layout.on_boarding_image
+import javax.inject.Inject
+
+
+class FingerprintFragment : DaggerFragment(), FingerprintView {
+
+  @Inject
+  lateinit var preferencesRepositoryType: PreferencesRepositoryType
+
+  companion object {
+    fun newInstance(): Fragment {
+      return FingerprintFragment()
+    }
+  }
+
+  private lateinit var presenter: FingerprintPresenter
+
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    presenter =
+        FingerprintPresenter(this, AndroidSchedulers.mainThread(),
+            Schedulers.io(), CompositeDisposable(), preferencesRepositoryType)
+  }
+
+
+  override fun showClickText() {
+    val toast = Toast.makeText(context, "IT WORKED", Toast.LENGTH_SHORT)
+    toast.show()
+  }
```

Comment:
Remove test/debug code

Reply:
Also removed getImageClick function because it was only needed for testing.